features/step_definitions/forum/category_steps.rb

```diff
@@ -30,6 +30,10 @@
 When /^I (follow|press) "([^"]*)" for category "([^"]*)"$/ do |action, label, name|
   element = action == 'follow' ? 'a' : 'button'
 
+  # FIXME!
+  # find(:xpath, %(//*[@id="categories"]//td[text()="#{name}"]/following-sibling::td//#{element}[text()="#{label}"]))
```

Comment:
Fixed in https://github.com/3scale/porta/pull/761/commits/6518aa916cb74fad34ffc73d8a0a7a9d331079f7 😄